dll/win32/comctl32/listview.c

```diff
@@ -8749,6 +8749,13 @@ static DWORD LISTVIEW_SetExtendedListViewStyle(LISTVIEW_INFO *infoPtr, DWORD mas
         LISTVIEW_UpdateScroll(infoPtr);
     }
 
+#ifdef __REACTOS__
+    if((infoPtr->dwLvExStyle ^ old_ex_style) & LVS_EX_SNAPTOGRID)
```

Comment:
Judging by this change, it also triggers when `LVS_EX_SNAPTOGRID` is removed, is that intentional?

Reply:
Good catch!  I'll get this fixed up later this evening.